src/app/components/task/SingleChoiceTask.js

```diff
@@ -164,8 +165,7 @@ class SingleChoiceTask extends Component {
               ))}
             </RadioGroup>
 
-            <div
-              style={{ display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}
+            <Box display="flex" justifyContent="flex-start" alignItems="center"
```

Comment:
Please break props into lines.

`168:18  error  Property should be placed on a new line             react/jsx-first-prop-new-line`
`168:33  error  Prop `justifyContent` must be placed on a new line  react/jsx-max-props-per-line`